### PR TITLE
Updates to the Croissant turtle definition to align with the spec, and…

### PR DESCRIPTION
**Pull request by benjelloun**
… introduce ML-specific constructs.

### CODE REVIEW

**Comment by benjelloun** [comment]
@marcenacp @ccl-core can you pls take a look at this PR? Thanks!

File: docs/croissant.ttl [hunk 1]
```diff
@@ -245,3 +255,37 @@ croissant:jsonQuery a rdf:Property ;
...
+
+croissant:SegmentationMask a rdf:class ;
+  rdfs:label "SegmentationMask" ;
+  rdfs:comment "A data type representing a segmentation mask." ;
+  rdfs:subClassOf schema:StructuredValue .
+
+croissant:BoundingBox a rdf:class ;
+  rdfs:label "Label" ;
+  rdfs:comment "A data type representing a bounding box." ;
+  rdfs:subClassOf schema:Text .
```
Comment (marcenacp) [inline docs/croissant.ttl:291]: `schema: StructuredValue` instead of `schema: Text`?

File: docs/croissant.ttl [hunk 2]
```diff
@@ -245,3 +255,37 @@ croissant:jsonQuery a rdf:Property ;
...
+  rdfs:comment "Test split." .
+
+croissant:ValidationSplit a croissant:Split ;
+  rdfs:label "ValidationSplit" ;
+  rdfs:comment "Validation split." .
+
+croissant:Label a rdf:class ;
+  rdfs:label "Label" ;
+  rdfs:comment "A machine learning label." ;
+  rdfs:subClassOf schema:Intangible .
```
Comment (marcenacp) [inline docs/croissant.ttl:281]: Can it still be used even if it's Intangible?

**Review by marcenacp — APPROVED** [review]
Thanks!

**Pull request merged**